hack/verify-api-reference-docs.sh

```diff
@@ -36,7 +36,7 @@ TMP_ROOT="${KUBE_ROOT}/_tmp"
 echo "diffing ${API_REFERENCE_DOCS_ROOT} against freshly generated docs"
 ret=0
 diff -NauprB -I 'Last update' --exclude=*.md "${API_REFERENCE_DOCS_ROOT}" "${OUTPUT_DIR}" || ret=$?
-rm -rf "${TMP_ROOT}"
+# rm -rf "${TMP_ROOT}"
```

Comment:
Ack. didn't mean to comment it out. I assume you're adding it back, let me know if you want me to instead.